docs/format/tile.rst

```diff
@@ -101,6 +110,8 @@ Interpreted as:
 tileconn.json
 -------------
 
+The file `tileconn.json` contains the information how the wires of neighboring tiles are connected to each other. It contains one entry for each pair of tile types, each containing a list of pairs of wires that belong to the same node.
```

Comment:
This is a good place to add the tile wire, pip, site pin diagram.

Reply:
Added a FIXME.